app/assets/javascripts/components/autocomplete.js

```diff
@@ -15,7 +15,7 @@ if ($autocompletes) {
     new accessibleAutocomplete.enhanceSelectElement({
       selectElement: $el,
       minLength: 3,
-      showNoOptionsFound: false,
+      defaultValue: '',
```

Comment:
Not sure why we're changing this?

Reply:
We want to show if there're no results found and we want to set the default for this field to an empty option. These changed make the preview of the components work as expected - which didn't until now.